tiflash/troubleshoot-tiflash.md

```diff
@@ -170,40 +170,23 @@ After deploying a TiFlash node and starting replication (by performing the ALTER
         }' <http://172.16.x.xxx:2379/pd/api/v1/config/rule>
     ```
 
-5. Check whether the connection between TiDB or PD and TiFlash is normal.
+5. Check whether TiDB has created `placement-rule` for tables.
```

Comment:
I think it may be better that 5's `placement-rule` and 6's `TiFlash placement rules` shared the same format.

Reply:
Fixed in 63a4d09